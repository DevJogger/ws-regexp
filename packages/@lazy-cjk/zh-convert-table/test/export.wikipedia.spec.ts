import _m0 = require('../wikipedia');
import _m1 from '../wikipedia';

test(`import check`, () =>
{

	let actual = Object.keys(_m0);
	let expected = Object.keys(_m1);

	expect(actual).toStrictEqual(expected);
	//expect(actual).toBeInstanceOf(Date);
	expect(actual).toMatchSnapshot();

});
