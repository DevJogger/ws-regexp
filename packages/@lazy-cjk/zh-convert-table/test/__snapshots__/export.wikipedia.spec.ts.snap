// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import check 1`] = `
Array [
  "wiki_s2t",
  "wiki_t2s",
  "wiki_s2t_v2",
  "wiki_t2s_v2",
  "default",
]
`;
